mlrun/__init__.py

```diff
@@ -160,3 +160,81 @@ def get_sample_path(subpath=""):
     if subpath:
         samples_path = path.join(samples_path, subpath.lstrip("/"))
     return samples_path
+
+
+def env_from_file(env_file, to_dict=False, set_env=True):
+    """Read and set and/or return environment variables from a file
+    the env file should have lines in the form KEY=VALUE, comment line start with "#"
+
+    example file::
+
+        # this is an env file
+        MLRUN_DBPATH=https://mlrun-api.default-tenant.app.xxx.iguazio-cd1.com
+        V3IO_USERNAME=admin
+        V3IO_API=https://webapi.default-tenant.app.xxx.iguazio-cd1.com
+        V3IO_ACCESS_KEY=MYKEY123
+        AWS_ACCESS_KEY_ID-XXXX
+        AWS_SECRET_ACCESS_KEY=YYYY
+
+    usage::
+
+        # set the env vars from a file + return the results as a dict
+        env_dict = mlrun.env_from_file(env_path, to_dict=True)
+
+    :param env_file: path/url to env file
+    :param to_dict:  set to True to return the env as a dict
+    :param set_env:  set to False to skip updating the current OS env
```

Comment:
This param also seems redundant to me, keep things simple, what sense there is in doing something like `set_env_from_file(set_env=False)` if you need it internally, have an internal function doing that for you, no need to expose it outside.
Also, IMO this function should just become a classmethod of the mlrun config - `mlrun.mlconf.from_env_file(file_path)` or `mlrun.mlconf.load_env_file(file_path)` or something like that

Reply:
assume i want to copy env file to set local env and/or set the function env (`fn.set_envs()`), i rather do it in one call

Reply:
also i think its an essential func (why have it under `mlrun.` ) and its not specific to conf (can use it to load S3 env, user env..)